rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -1046,4 +1045,14 @@ bool RSocketStateMachine::ensureOrAutodetectFrameSerializer(
   frameSerializer_ = std::move(serializer);
   return true;
 }
+
+size_t RSocketStateMachine::getConsumerAllowance(StreamId streamId) const {
+  size_t consumerAllowance = 0;
+  auto it = streamState_.streams_.find(streamId);
+  if (it != streamState_.streams_.end()) {
```

Comment:
Nit: `return it != streamState_.streams_.end() ? it->second->getConsumerAllowance() : 0;`